app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -41,7 +41,9 @@ def unmute
   end
 
   def transcript
-    ConversationReplyMailer.conversation_transcript(@conversation, params[:email])&.deliver_later if params[:email].present?
+    if params[:email].present?
```

Comment:
Should we send an unprocessable entity response if the email param is not present?